.quickstart/quickstart.yml

```diff
@@ -23,4 +23,13 @@ destination_configurations:
   databricks:
     dispatch:
       - macro_namespace: dbt_utils
-        search_order: [ 'spark_utils', 'dbt_utils' ]
\ No newline at end of file
+        search_order: [ 'spark_utils', 'dbt_utils' ]
+public_models: [
+  "salesforce__contact_enhanced",
+  "salesforce__daily_activity",
+  "salesforce__manager_performance",
+  "salesforce__opportunity_enhanced",
+  "salesforce__opportunity_line_item_enhanced",
+  "salesforce__owner_performance",
+  "salesforce__sales_snapshot"
+]
```

Comment:
@fivetran-joemarkiewicz Do we want to include the [history models](https://github.com/fivetran/dbt_salesforce/tree/main/models/salesforce_history) here?

Reply:
Good question @fivetran-catfritz. The history models are not available in Quickstart. So we will intentionally leave them off the list.